src/client/datascience/insidersNativeNotebookSurveyBanner.ts

```diff
@@ -63,39 +66,43 @@ export class InsidersNativeNotebooksSurveyBanner implements IJupyterExtensionBan
 
     private readonly showBannerState: IPersistentState<ShowBannerWithExpiryTime>;
 
-    private readonly surveyLink: string;
+    private readonly surveyLink: string = 'https://aka.ms/vscjupyternb';
```

Comment:
No need to change, however for somethign like this a constant is better. We don't really need a property.